index.html

```diff
@@ -8,51 +8,54 @@
 
-<div class="quicktube" data-quicktube="kittens">
-    <div class="quicktube__video quicktube__video--paused" data-quicktube-video="k6QanQUaDOo">
-    </div>
-    <div data-quicktube-play="kittens"  class="quicktube__poster" data-quicktube-poster>
-        <div class="quicktube__play quicktube__btn">
-            Play
+<div class="page-container">
+    <div class="quicktube" data-quicktube="gneBUA39mnI">
+        <div class="quicktube__video quicktube__video--paused" data-quicktube-video></div>
+        <div data-quicktube-play  class="quicktube__poster" data-quicktube-poster>
+            <div class="quicktube__play quicktube__btn" tabindex="0">
+                Play
+            </div>
         </div>
     </div>
-</div>
-
-<!-- /HTML you need -->
 
-<!-- Demonstration -->
+    <div class="quicktube" data-quicktube="QFmLReorTSY">
+        <div class="quicktube__video quicktube__video--paused" data-quicktube-video></div>
+        <div data-quicktube-play  class="quicktube__poster" data-quicktube-poster>
+            <div class="quicktube__play quicktube__btn" tabindex="0">
+                Play
+            </div>
+        </div>
+    </div>
 
-<div data-show-while-playing class="text--center" style="display: none;">
-    <p class="quicktube__btn quicktube__btn--stop" data-quicktube-stop="kittens">
-        Stop the vid!
-    </p>
-</div>
-<div data-show-while-playing style="display: none; padding-left: 1em; padding-right: 1em;">
-    <p>
-        Oh, hey... you can trigger stuff to happen when the video starts/stops via:
-        <code>
-            $(window).on("quicktube:play", function(quicktubeId, $quicktubeEl) {
-                // do something
-            });
-        </code>
-    </p>
+    <div data-show-while-playing class="u-text-center" style="display: none;">
+        <p class="quicktube__btn quicktube__btn--stop" data-quicktube-stop="gneBUA39mnI" tabindex="0">
+            Stop first video!
+        </p>
+        <p class="quicktube__btn quicktube__btn--stop" data-quicktube-stop="QFmLReorTSY" tabindex="0">
+            Stop second video!
+        </p>
+    </div>
 </div>
 
-<!-- /Demonstration -->
+<!-- /HTML you need -->
 
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
 <script src="quicktube.js"></script>
 <script>
+
     // Remember to initialise QuickTube at some point in your app
-    $(document).ready(function() {
-        // Demonstration...
-        $(window).on("quicktube:play", function() {
-            $("[data-show-while-playing]").show();
-        });
-
-        $(window).on("quicktube:pause", function() {
-            $("[data-show-while-playing]").hide();
-        });
+    document.addEventListener('DOMContentLoaded', function() {
+        const showingItem = document.querySelector('[data-show-while-playing]');
+
+        quicktubeController();
+
+        window.addEventListener("quicktube:play", function() {
+            showingItem.style.display = 'block';
```

Comment:
Just there as an example and it didn't work using ()=>{}

These added event listeners may well get removed, its usefulness needs evaluation :)